warehouse/pom.xml

```diff
@@ -460,7 +460,7 @@
                         <threadCountSuites>0</threadCountSuites>
                         <!-- BT_ROOT is for the tests that use basis tokenization -->
                         <argLine>-XX:+TieredCompilation -XX:TieredStopAtLevel=1  -DBT_ROOT=${env.BT_ROOT} -Dfile.encoding=UTF8 -Duser.timezone=GMT -Xmx2g -Djava.library.path=${env.HADOOP_HOME}/lib/native -Dapple.awt.UIElement=true -Djava.security.krb5.realm= -Djava.security.krb5.kdc=
-                            --add-opens java.xml/jdk.xml.internal=ALL-UNNAMED</argLine>
+                            --add-opens java.xml/jdk.xml.internal=ALL-UNNAMED @{argLine}</argLine>
```

Comment:
What is the @{argLine} property adding here?

Reply:
Jacoco adds things to the argLine property, so it needs to be in there like that.